workflows/npm-package-release.yml

```diff
@@ -0,0 +1,61 @@
+name: NPM Package Release
+
+on:
+  workflow_call:
+        inputs:
+            node_version:
+                description: 'Which version of node to use for build'
+                type: string
+                required: false
+                default: '16.x'
```

Comment:
Do we want a newer version of node? Perhaps what's used in the current WP release